dash/webapp/next.config.js

```diff
@@ -0,0 +1,10 @@
+module.exports = {
+  exportPathMap: async function (defaultPathMap) {
```

Comment:
This creates subfolders in the `out` folder, ensuring pa11y.html become `pa11y/index.html`.
Exporting those does not overwrite the other files within `pa11y/` we still need, such as `report.json`.

Reply:
I had to tweak it from the suggested solution (header links pointing to `pa11y.html`) as it wouldn't work when running locally.